src/components/pagination.module.scss

```diff
@@ -0,0 +1,52 @@
+.pagination {
+  display: grid;
+  grid-column-start: 1;
+  grid-column-end: 2;
+  width: 100%;
+  background-color: var(--background-pink);
+
+  &__list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+  }
+
+  &__item,
+  &__current-item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 1.5rem;
+    height: 1.5rem;
+    margin: 4px;
+    background-color: var(--background-pink);
+    border: 1px solid var(--main-pink);
+    cursor: pointer;
+  }
+
+  &__current-item {
```

Comment:
change it to BEM modifier like `&__item--current` and use @extend inside 
https://sass-lang.com/documentation/at-rules/extend